Art/Source/3D/Sequences/compose.py

```
@@ -0,0 +1,328 @@
+                            math.radians(focus["rotation_deg"][1]),
+                            math.radians(focus["rotation_deg"][2]),
+                        ))
+
+                current_time = end_time + self.transition_time
+            else:
+                current_time = end_time
+
+        # Smooth all keyframes
+        print("Smoothing keyframes...")
+        from common.core import smooth_all_keyframes
+        smooth_all_keyframes = getattr(
```

Ruff (0.14.7)

Redefinition of unused `smooth_all_keyframes` from line 181

(F811)